lib/screens/v2/candidate_generator/dup.ex

```diff
@@ -0,0 +1,78 @@
+defmodule Screens.V2.CandidateGenerator.Dup do
+  @moduledoc false
+
+  alias Screens.Config.Screen
+  alias Screens.Config.V2.Dup
+  alias Screens.Config.V2.Header.CurrentStopId
+  alias Screens.Stops.Stop
+  alias Screens.V2.CandidateGenerator
+  alias Screens.V2.Template.Builder
+  alias Screens.V2.WidgetInstance.{NormalHeader, Placeholder}
+
+  @behaviour CandidateGenerator
+
+  @impl CandidateGenerator
+  def screen_template do
+    {:screen,
+     %{
+       screen_normal: [
+         {:rotation_zero,
+          %{
+            body_normal_zero: [
+              :header,
+              :main_content_primary,
+              :inline_alert_zero
+            ],
+            screen_takeover_zero: [:full_screen_zero]
+          }},
+         {:rotation_one,
+          %{
+            body_normal_one: [
+              :header,
+              :main_content_primary
```

Comment:
At the time, I did the share slot names mostly because I thought it was convenient. I'm with you now, I don't think we should rely on this working forever. Every slot name should have a unique name now.